File: elza-core/src/main/java/cz/tacr/elza/controller/vo/ap/item/ApUpdateItemVO.java
package cz.tacr.elza.controller.vo.ap.item;

import cz.tacr.elza.controller.vo.nodes.descitems.UpdateOp;

/**
 * Položka změny.
 */
public class ApUpdateItemVO {

    /**
     * Typ změny.
     */
    private UpdateOp updateOp;

    /**
     * Položka změny.
     */
    private ApItemVO item;

    public UpdateOp getUpdateOp() {
        return updateOp;
    }

    public void setUpdateOp(final UpdateOp updateOp) {
        this.updateOp = updateOp;
    }

    public ApItemVO getItem() {
        return item;
    }

    public void setItem(final ApItemVO item) {
        this.item = item;
    }
}

File: elza/elza-react/src/api/ApUpdateItemVO.ts
import {ApItemVO} from "./ApItemVO";
import {UpdateOp} from "./UpdateOp";

export interface ApUpdateItemVO {
    /**
     * Typ změny.
     */
    updateOp: UpdateOp;

    /**
     * Položka změny.
     */
    item: ApItemVO;
}
